vulcanai/models/metrics.py

```diff
@@ -1012,3 +1014,85 @@ def cross_validate(network, data_loader, k, epochs,
             logger.info(all_results)
             return all_results
 
+    def conduct_sensitivity_analysis(self, network, data_loader, filename,
+                                     features, cutoff=20):
+        """
+        Will conduct tests to figure out directionality of features by finding all the unique feature values present in
+        the dataset and setting the feature for all examples to every unique value of that feature.
+
+        Parameters
+        ----------
+            network : BaseNetwork
+                Network descendant of BaseNetwork.
+            data_loader : DataLoader
+                A DataLoader object which contains all the data
+            filename: string
+                The name of the file to save the test results to
+            cutoff: int
+                Maximum number of unique feature values for a particular feature that will be tested
+
+        Returns
+        -------
+        #TODO
+        """
+
+        if network._num_classes is None or network._num_classes == 0:
+            raise ValueError('There\'s no classification layer')
+
+        dataset = data_loader.dataset
+
+        test_df = None
+        col_headers = None
+
+        for feature in range(len(features)):
```

Comment:
would `idx` (aka index) be a better name than feature if you are iterating the the index locations of the features instead of the features themselves? or something like feature_idx?